doc/Tags-and-Mentions.md

```diff
@@ -21,11 +21,19 @@ You can tag a person on a different network or one that is **not in your social
 	
 * @mike@macgirvin.com - This is called a "remote mention" and can only be an email-style locator, not a web URL.
 
-Unless their system blocks unsolicited "mentions", the person tagged will likely receive a "Mention" post/activity or become a direct participant in the conversation in the case of public posts. Please note that Friendica blocks incoming "mentions" from people with no relationship to you. This is a spam prevention measure.
+Unless their system blocks unsolicited "mentions", the person tagged will likely receive a "Mention" post/activity or become a direct participant in the conversation in the case of public posts.
+Please note that Friendica blocks incoming "mentions" from people with no relationship to you.
```

Comment:
I referred to "please note that Friendica blocks incoming "mentions" from people with no relationship to you."

I don't think that we are blocking that. Especially for OStatus I would deny that behaviour.